src/images/icons/icon-smiley-wink.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
We're using `hm-pattern-library/gulp/gulp-svg.js` to minify these files, so the extra noise from Adobe shouldn't be a problem.